deploy/osd-scan-tester/config.yaml

```diff
@@ -0,0 +1,7 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet:
+  matchExpressions:
+    - key: api.openshift.com/fedramp
+      operator: In
+      values:
+        - "true"
```

Comment:
This means it would install to all clusters. Similar to ids-tester, change to only non-production: https://github.com/openshift/managed-cluster-config/blob/9916ac4591ea4dcfe9c12287360d1c3f3bb2a44b/deploy/osd-ids-tester/config.yaml#L8-L11

Reply:
+1 to make this change for non-production first.  Change control tracking will be required for production, and this could be the proof of testing before production release.